### PR TITLE
RedHat 8 support

### PR DESCRIPTION
**Pull request by kbcz1989**
Hi,

I nedded rh8 support so I added it. Can you please check?

BR

### CODE REVIEW

File: tasks/install_dnf.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   - block:
       - name: PostgrSQL | Install all the required dependencies | dnf
         dnf:
-          name: "ca-certificates, python-pycurl, glibc-common, libselinux-python, python*-psycopg2"
```
Comment (dmlb2000) [inline tasks/install_dnf.yml:9]: This bit concerns me, did you check that this works with Fedora?

**Comment by dmlb2000** [comment]
I also got an idempotence failure for tasks

 * [centos8] => postgresql : PostgrSQL | Install all the required dependencies | dnf
 * [centos8] => postgresql : disable system postgresql

https://travis-ci.org/github/pacifica/ansible-pacifica/jobs/677989400

**Comment by taybin** [comment]
Any updates on this?

**Comment by gclough** [comment]
@kbcz1989, thanks for the Pull Request.  The CI pipeline has been rebuilt since you did this, and there are a couple of merge conflicts.  Could you rebase and send a new commit, which will trigger the pipeline and prove it works on all other platforms?

**Comment by MrMegaNova** [comment]
RH8 is now supported

**Pull request closed without merging**